backend/services/user_groups.go

```diff
@@ -76,33 +76,31 @@ func (i CreateUserGroupInput) validateUserGroupInput() error {
 	return nil
 }
 
-func AddUsersToGroup(tx database.ConnectionProxy, userSlugs []string, groupID int64) error {
-	for _, userSlug := range userSlugs {
-		var userID int64
-		err := tx.Get(&userID, sq.Select("id").From("users").Where(sq.Eq{"slug": userSlug}))
-		if err != nil {
-			return backend.WrapError("Unable to get user id from slug", backend.BadInputErr(err, fmt.Sprintf(`No user with slug %s was found`, userSlug)))
+func AddUsersToGroup(tx *database.Transactable, userSlugs []string, groupID int64) error {
+	if len(userSlugs) > 0 {
```

Comment:
@JoelAtDeluxe Initially I improved this function by iterating through each userSlug and performing an upsert, but here's my attempt to reduce it to one query. It looks a little janky, but (in theory) should be much more performant that inserting each userSlug individually. Any way you'd improve this?